packages/create-cloudflare/templates/do-hello-world/js/src/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Welcome to Cloudflare Workers! This is your first Durable Objects application.
+ *
+ * - Run `npm run dev` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your Durable Object in action
+ * - Run `npm run deploy` to publish your application
+ *
+ * Learn more at https://developers.cloudflare.com/durable-objects
+ */
+
+export class MyDurableObject {
+	constructor(state, env) {}
+
+	async fetch(request) {
+		return new Response("Hello World");
+	}
+}
+
+export default {
+	async fetch(request, env, ctx) {
+		let id = env.MY_DURABLE_OBJECT.idFromName(new URL(request.url).pathname);
```

Comment:
nit: It's easy for our trained eyes to distinguish what matters, but I think we ought to emphasize `idFromName()` here.
```
let name = new URL(request.url).pathname;
// We will create a Durable Object ID using the pathname.
// This refers to a unique instance of our 'MyDurableObject' class above.
let id = env.MY_DURABLE_OBJECT.idFromName(name);
```